src/components/input-group/daikin-input-group.ts

```diff
@@ -127,6 +129,16 @@ type HelperType = MergeVariantProps<typeof cvaHelper>["type"];
  * </daikin-input-group>
  * ```
  *
+ * With Checkbox Group:
+ *
+ * ```html
+ * <daikin-input-group>
+ *   <daikin-checkbox-group>
+ *     <daikin-checkbox name="Name of Checkbox Group" value="Value of Checkbox Group" label="Label Text"></daikin-checkbox>
```

Comment:
- Usually name and value do not contain whitespace.
- We should have more than one checkbox for checkbox group
- Should we include `name` or not? (I'm not sure.)

```suggestion
 *     <daikin-checkbox name="name1" value="value1" label="Checkbox Item 1"></daikin-checkbox>
 *     <daikin-checkbox name="name2" value="value2" label="Checkbox Item 2"></daikin-checkbox>
 *     <daikin-checkbox name="name3" value="value3" label="Checkbox Item 3"></daikin-checkbox>

Reply:
> Should we include name or not? (I'm not sure.)

@yodas7 
I think `name` should be needed. when submit from a form, the data I expect should be multiple name&value pairs